skimage/registration/_masked_phase_cross_correlation.py

```diff
@@ -130,6 +130,9 @@ def cross_correlate_masked(arr1, arr2, m1, m2, mode='full', axes=(-2, -1),
         maximum translation, while a higher `overlap_ratio` leads to greater
         robustness against spurious matches due to small overlap between
         masked images.
+    wrap_axes: tuple of ints, optional
+        Axes along which to not zero-pad `arr1` and `arr2` causing the
+        correlation to wrap.

```

Comment:
Just a suggestion to explicitly indicate the output shape to expect when an axis is marked as periodic.

```suggestion
        correlation to wrap. The shape of the output will match the shape of
        `arr1` along any axis included in `wrap_axes`.
```